python-backend/app/api/variances/resources/variance_resource.py

```diff
@@ -43,24 +60,70 @@ def get(self):
             'total': pagination_details.total_results,
         }
 
+    @classmethod
+    def _build_filter(cls, model, field, op, argfield):
+        return {
+            'model': model,
+            'field': field,
+            'op': op,
+            'value': argfield
+        }
+
+    def _apply_filters_and_pagination(self, args):
 
-    def _apply_filters_and_pagination(self,
-                                      page_number=PAGE_DEFAULT,
-                                      page_size=PER_PAGE_DEFAULT,
-                                      application_status=None):
         status_filter_values = list(map(
             lambda x: x.variance_application_status_code,
             VarianceApplicationStatusCode.active()))
 
-        if application_status is not None:
-            status_filter_values = application_status.split(',')
+        conditions = []
+        if args["application_status"] is not None:
+            status_filter_values = args["application_status"].split(',')
+            conditions.append(self._build_filter('Variance' , 'variance_application_status_code', 'in',  status_filter_values))
```

Comment:
FYI you can also do these filtering using native sql-alchemy provided functions directly on the model like in here, just to keep everything following the same pattern.
https://github.com/bcgov/mds/pull/936/files#diff-c3f49332dad6bb23875dae032ba7ea71R67

I do see the file already was using the filter helper method so i guess we dont have a hard set pattern for this kind of stuff yet so maybe that should be something to talk about 🤔